core/modules/wsched/BlendScheduler.cc

```diff
@@ -417,6 +417,21 @@ void BlendScheduler::_logChunkStatus() {
 }
 
 
+nlohmann::json BlendScheduler::statusToJson() {
+    nlohmann::json status;
+    status["name"] = getName();
+    status["priority"] = getPriority();
+    status["num_tasks_in_queue"] = getSize();
+    status["num_tasks_in_flight"] = getInFlight();
+    nlohmann::json schedulers = nlohmann::json::array();
+    for (auto&& sched: _schedulers) {
```

Comment:
Is `auto&&` better than `auto const&` ?

Reply:
I tend to use `auto&&` as it works in all scenarios regardless of 1) the `iterable` type, 2) its `const`-ness, and 3) the `const`-ness of a method where the `for` loop is used. My understanding is that `auto&&` would have the sam effect as `auto const&` in this context.